common/xml/xmlsort.xsl

```diff
@@ -4,51 +4,138 @@
   <xsl:output method="xml" indent="yes"/>
   <xsl:strip-space elements="*"/>
 
+  <!-- template-function: Allow from attribute which gives you a relative to a given pb_type -->
```

Comment:
Can we make a PR just for the XML sort changes?  XSLT is fairly inscrutable language, and so I'd like to see specific tests for the behaviors changed here, and it will be easier to review this particular change in isolation.

Reply:
This was done.